src/app/zap-templates/override.js

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+function atomicType(arg) {
+  switch (arg.name) {
+  case 'attribute_id':
+    return 'chip::AttributeId';
+  case 'cluster_id':
+    return 'chip::ClusterId';
+  default:
+    throw 'not overriding';
+  }
+}
+
+function nonAtomicType(arg) {
+  switch (arg.name) {
```

Comment:
To be honest I was hoping to use `{{asUnderlyingZclType}}` instead of having to use `{{asUnderlyingType}}`. So my plan was to switch to `{{asUnderlyingZclType}}` while it will be ready and remove the usage of `nonAtomic` as I agree it is weak.

Thoughts ?

Reply:
I end up removing it and updating the `zap/repo`.